packages/nc-gui/components/general/EmojiIcons.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+import { Icon } from '@iconify/vue'
+import InfiniteLoading from 'v3-infinite-loading'
+import { emojiIcons } from '#imports'
+
+const emit = defineEmits(['selectIcon'])
+let search = $ref('')
+
+let toIndex = $ref(60)
```

Comment:
need some comments explaining what `toIndex` means and why the initial value is `60`.